src/modules/commander/state_machine_helper.h

```diff
@@ -68,6 +68,32 @@ enum class link_loss_actions_t {
 	LOCKDOWN = 6,		// Kill the motors, same result as kill switch
 };
 
+enum class offboard_loss_actions_t {
```

Comment:
I would not use `_t` as the suffix here because that's usually used for `typedef`.

What about `OffboardLossActions` instead?

Reply:
I am fine with this naming format, I was just being coherent with `link_loss_actions_t` above.

Is there an official PX4 naming convention that I can check?

Reply:
I don't think there is but you're right there should be. It's a bit a mess of historical and newer styles.

For now you can leave it consistent with the rest and we can change it later.